src/components/Heating/OptionsMenu.tsx

```diff
@@ -0,0 +1,136 @@
+import { useMemo, useState } from 'react';
+import { useHandleGestures } from '../../hooks/useHandleGestures';
+import { styled } from 'styled-components';
+import { useProfileContext } from '../../context/ProfileContext';
+import { useAppDispatch } from '../store/hooks';
+import { setScreen } from '../store/features/screens/screens-slice';
+
+const OPTION_HEIGHT = 38;
+
+const MenuContainer = styled.div<{ $num_options: number }>`
+  display: flex;
+  flex-direction: column;
+  height: ${({ $num_options }) => $num_options * OPTION_HEIGHT}px;
+  overflow: hidden;
+  margin-bottom: 20px;
+
+  &.fade-options-enter {
+    opacity: 0;
+    height: 0;
+    margin-bottom: 0px;
+    margin-top: -20px;
+  }
+  &.fade-options-exit {
+    opacity: 1;
+    height: ${({ $num_options }) => $num_options * OPTION_HEIGHT}px;
+    margin-top: 0;
+    margin-bottom: 20px;
+  }
+  &.fade-options-enter-active {
+    opacity: 1;
+    height: ${({ $num_options }) => $num_options * OPTION_HEIGHT}px;
+    margin-top: 0;
+    margin-bottom: 20px;
+  }
+  &.fade-options-exit-active {
+    opacity: 0;
+    height: 0;
+    margin-top: -20px;
+    margin-bottom: 0px;
+  }
+  &.fade-options-enter-active,
+  &.fade-options-exit-active {
+    transition: all 600ms;
+    overflow: hidden;
+  }
+`;
+
+const MenuEntry = styled.div<{ $active?: boolean }>`
+  min-width: 250px;
+  height: ${OPTION_HEIGHT}px;
+  flex-shrink: 0;
+  border-radius: 4px;
+  background: ${({ $active }) => ($active ? '#F5C444' : 'transparent')};
+  color: ${({ $active }) => ($active ? '#000000' : '#FFFFFF')};
+
+  line-height: 1;
+  letter-spacing: 0;
+  padding-left: 20px;
+  font-family: 'ABC Diatype';
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: 200%;
+  letter-spacing: 4px;
+  text-transform: uppercase;
+`;
+
+export const OptionsMenu = ({
+  ignoreGestures
+}: {
+  ignoreGestures: boolean;
+}) => {
+  const { localProfile, setDetailsProfileSelected } = useProfileContext();
+  const [activeIndex, setActiveIndex] = useState(0);
+  const [autostart, setAutostart] = useState(false);
+  const dispatch = useAppDispatch();
+
+  const OPTIONS = useMemo(
+    () => [
+      ...(localProfile?.display?.description
+        ? [
+            {
+              key: 'description',
+              label: 'Description'
+            }
+          ]
+        : []),
+      {
+        key: 'edit',
+        label: 'Edit profile'
+      },
+      {
+        key: 'auto_start',
+        label: autostart ? 'Auto brew' : 'Push to brew'
```

Comment:
workflow-wise, is this adding the ability to:
* have a default "auto-brew" set to "manual" in the config menu
* but when brewing a profile, once you have locked in the portafilter, you can toggle it to "auto-brew" ?  (or choose to have it be "manual" still)

This would fix the workflow issue where i don't want it to brew before i lock in the portafilter, but happy to do so (on some profiles) after it gets to temp.

Reply:
notes:  implementation is changing a bit, but answer is "yes".  intention is to enable a user to have a default setting for "auto-start", and override it on a specific brew.

in my particular case, i'd likely set it to "manual start", and on some profiles, once i've locked in the portafilter, would toggle it to "auto-start" as soon as it's up-to-temp.